src/main/java/com/amazon/ion/impl/bin/IonRawBinaryWriter_1_1.kt

```diff
@@ -859,12 +859,12 @@ class IonRawBinaryWriter_1_1 internal constructor(
                     thisContainerTotalLength++
                 } else if (isTagless) {
                     // End tagless group -- write the number of expressions, end with FlexUInt 0
-                    thisContainerTotalLength += writeCurrentContainerLength(lengthPrefixPreallocation)
+                    thisContainerTotalLength += writeCurrentContainerLength(maxOf(1, lengthPrefixPreallocation))
```

Comment:
This is now symmetrical with `stepIn` for this case, which always reserves at least one byte for the length. Without this fix, the byte is reserved in the buffer *and* a patch point is added, resulting in an erroneous extra byte in the final output. The same can be said for the other change in this file.

This change is verified by the added `taglessExpressionGroup` test.